.vscode/launch.json

```diff
@@ -4,6 +4,18 @@
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
   "version": "0.2.0",
   "configurations": [
+
+    {
+      "name": "(Windows) Launch",
+      "type": "cppvsdbg",
+      "request": "launch",
+      "program": "${workspaceFolder}/build_default_relwithdebinfo/bin/Orbit",
```

Comment:
I think instead of `${workspaceFolder}/build_default_relwithdebinfo}`, `${command:cmake.buildDirectory}` might work better.

Reply:
But this needs to the cmake-extension and might also not be appropriate for your use case.

Reply:
Ref: https://github.com/microsoft/vscode-cmake-tools/pull/726

Reply:
Just substituting this naively results in an error for me: "command cmake.buildDirectory not found". Not sure what else I need to do to make this work?

Reply:
Ah, didn't see your other replies before replying. @antonrohr Let me know what you think we should do here.

Reply:
You're probably not using the cmake extension. In this case it's fine for me to go with your solution. Personally I'm not a big fan of having a `.vscode` folder in the repo at all, but I'm also not a frequent user of vscode. Anton should have a look.

Reply:
I think `${workspaceFolder}/build_default_relwithdebinfo` is fine since its pretty much the standard way of doing that. I don't even know how the cmake extension would decide which build target I want to choose if I have multiple

Reply:
I don't have an opinion on this, I just want my debugger to work out-of-the box (and I'm just using this one target anyway). Can either of you approve for now? We can always change / improve this later if we find a better way.